gceletter/util.py

```diff
@@ -98,6 +111,11 @@ def validate_data(request, cls):
     return data
 
 
+def validate_request_data(request, cls):
```

Comment:
why did you break this function up? just curious if it's for readability or preferred convention reasons

Reply:
I did it for mainly for ease of testing - so I could test just the validation of some sample data without having to do an api request first. Also I think this might make it easier if we want to have a different pydantic model for a new endpoint we can reuse this function to help with the error handling. hopefully we can eventually pull a lot of these sorts of util functions out and reuse them across GCE, GCELETTER, etc. since I think now there's a lot of copy/paste between them.